frontend/elements/src/entry/home/sections/contactus-section.ts

```diff
@@ -0,0 +1,71 @@
+import { MEDIA_UI } from '@utils/path';
+import "@elements/entry/home/TOSORT/column-list";
+import "@elements/entry/home/social-networks";
+import { LitElement, html, css, customElement, property } from 'lit-element';
+@customElement('contactus-section')
+export class _ extends LitElement {
+  static get styles() {
+    return [css`
+    h3{
+      font-family: Poppins;
+      font-size: 20px;
+      font-weight: 800;   
+      color:#ffffff;
+    
+    }
+   main{
+     
+   }
+   
+   ul{
+    list-style-type: none;
+    margin:0;
+    padding:0;
+   }
+
+ 
+
+  .socialnetworks{
+     display:block;
+     margin-top:30px;
+  }
+
+  .list{
+    display:block;
+    margin-top:8px;
+ }
+  
+
+    `];
+  }
+
+
+ 
+ 
+
+  render() {
+
+    const {} = this;
+
+    return html`
+    <main>
+    <h3> Contact us </h3>
+    <ul>
+    <column-list text_line="info@jewishinteractive.org" color="white" class="list"></column-list>
```

Comment:
should be separated into STR_ and outside of the render function (for all these items)

keep in mind that it is not only prep for copywriting, but also internationalization - e.g. we'll need `Tel` in Hebrew.